src/js/components/replenishment/CreateReplenishment.jsx

```diff
@@ -145,17 +145,22 @@ const FIELD = {
 function validate(values) {
   const errors = {};
   errors.requirements = [];
-
   _.forEach(values.requirements, (item, key) => {
-    if (item.quantity) {
-      if (item.quantity < 1) {
-        errors.requirements[key] = { quantity: 'react.replenishment.error.quantity.label' };
-      }
-      if (item.quantity > item.quantityAvailable) {
-        errors.requirements[key] = { quantity: 'react.replenishment.error.quantity.greaterThanQATP.label' };
+    // error shouldn't be added to fields which are not checked
+    if (item.checked) {
+      if (item.quantity) {
+        if (item.quantity < 1) {
```

Comment:
I would do this as:
```
if (item.quantity && item.quantity < 1) {...}
else if (item.quantity && item.quantity > item.quantityAvailable) {...}
else if (!item.quantity && item.checked) {...}
```
or 
```
if (item.quantity) {
   if (item.quantity < 1) { ... }
   if (item.quantity > item.quantityAvailable) { ... }
} else {
   if (item.checked) { ... }
}
```
I think that would look a bit better than
```
if () {
    if () {
        if () {   
...
```

Reply:
if I can propose:
We don't have to do nested `ifs` for 
```
if (item.quantity) {
  if (item.quantity < 1) {
    ...
  }
}
```
we could simplify it to `if (item?.quantity < 1)`